app/serializers/artist_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class ArtistSerializer
+  include JSONAPI::Serializer
+
+  attributes :id, :name
+end
```

Comment:
Try to be consistent with newlines at the end of the file - either all have or all do not.